unit_tests/R7LoggerTests.php

```diff
@@ -1,7 +1,7 @@
 <?php
 	use PHPUnit\Framework\TestCase;
 
-	class R7LoggerTest extends TestCase
+	class R7LoggerTests extends TestCase
```

Comment:
iirc singular is convention. i don't particularly care but i don't see why we're changing this.

Reply:
Has to match the file name.
phpunit errors out otherwise